subt/artf_reporter.py

```diff
@@ -37,21 +38,74 @@ def __init__(self, config, bus):
         self.artf_xyz = []
         self.artf_xyz_accumulated = []  # items are [type, position, source, flag]
                                         # position = [x, y, z], flag(scored) = True/False/None
+        self.verbose = False
 
     def publish_artf(self, artf_xyz):
         count = 0
-        for artf_type, pos, src, scored in artf_xyz:
+        for artf_type, pos, src, scored in self.group_artf_for_report(artf_xyz):
             if scored is None:
```

Comment:
This is always true.

Reply:
True, true - I would probably rename the function as it is no longer "grouping"